.github/workflows/ci.yml

```diff
@@ -173,6 +172,11 @@ jobs:
     needs:
       - fmt
       - check-file-change
+    if: >
+      needs.check-file-change.outputs.parachain_src == 'true' ||
+      needs.check-file-change.outputs.parachain_test == 'true' ||
+      needs.check-file-change.outputs.tee_src == 'true' ||
+      needs.check-file-change.outputs.tee_test == 'true'
```

Comment:
same as `tee-build`, we don't need the `if` actually

Reply:
come to alignment after offline discussion.